lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -892,22 +1284,13 @@ def upload_reads(self, ctx, params):
         file_valid = self.validateFASTQ({}, [{'file_path': actualpath,
                                         'interleaved': interleaved}])
         if not file_valid[0][0]['validated']:
-            validation_error_message = "Invalid FASTQ file - Path: " + actualpath + "."
-            if shock:
-                if revid:
-                    validation_error_message += (
-                        " Input Shock IDs - FWD Shock ID : " +
-                        fwdid + ", REV Shock ID : " + revid +
-                        ". FWD File Name : " + fwdname +
-                        ". REV File Name : " + revname +
-                        ". FWD Path : " + fwdpath +
-                        ". REV Path : " + revpath + ".")
-                else:
-                    validation_error_message += (" Input Shock ID : " + fwdid +
-                                                 ". File Name : " + fwdname + ".")
-            elif revpath:
-                validation_error_message += (" Input Files Paths - FWD Path : " +
-                                             fwdpath + ", REV Path : " + revpath + ".")
+            validation_error_message = self._generate_validation_error_message(
```

Comment:
hmmm I did this because I no longer want to treat fwdid as fwdurl/fwdstaging. I think it's more confusing to me.